README.md

```diff
@@ -413,8 +413,12 @@ https://github.com/StructuralAnalysisFormat/StructuralAnalysisFormat-SDK
 
 ## Units.NET on other platforms
 
-The powerful strong-typed unites based on Units.NET [unite definition](/Common/UnitDefinitions) is available on other platforms!
+The powerful strong-typed unites based on Units.NET [unit definitions](/Common/UnitDefinitions) is available on other platforms!
 
-- JavaScript / TypeScript
+- ##### JavaScript / TypeScript
   - Package on NPM registery https://www.npmjs.com/package/unitsnet-js
   - Generate units source-code https://github.com/haimkastner/unitsnet-js
+
+- ##### Python
+  - Package on PyPI registery https://pypi.org/project/unitsnet-py
+  - Generate units source-code https://github.com/haimkastner/unitsnet-py
```

Comment:
How about this format:

```md
Get the same strongly typed units on other platforms, based on the same [unit definitions](/Common/UnitDefinitions).

| Platform                 | Name        | Links                                                                                                    | Maintainers  |
|--------------------------|-------------|----------------------------------------------------------------------------------------------------------|--------------|
| JavaScript<br>TypeScript | unitsnet-js | [npm](https://www.npmjs.com/package/unitsnet-js)<br>[github](https://github.com/haimkastner/unitsnet-js) | @haimkastner |
| Python                   | unitsnet-py | [pypi](https://pypi.org/project/unitsnet-py)<br>[github](https://github.com/haimkastner/unitsnet-py)     | @haimkastner |
```

Reply:
I dig that you are able to reuse the definitions for multiple other languages, increasing the value of the effort that was put in and hopefully getting more contributors on the units. Very cool 🙌

Reply:
Thank you!
And I'm really impressed by the effort put into the definitions to include so many unit systems, while also keeping them in the JSON format and ensuring they're easy to be consumed in other environments.

Regarding your suggestion, It looks good to me, and I will push a new commit with it within the next few minutes.